src/realm.h

```diff
@@ -3153,8 +3162,6 @@ RLM_API realm_sync_config_t* realm_sync_config_new(const realm_user_t*, const ch
 RLM_API realm_sync_config_t* realm_flx_sync_config_new(const realm_user_t*) RLM_API_NOEXCEPT;
 RLM_API void realm_sync_config_set_session_stop_policy(realm_sync_config_t*,
                                                        realm_sync_session_stop_policy_e) RLM_API_NOEXCEPT;
-RLM_API void realm_sync_config_set_error_handler(realm_sync_config_t*, realm_sync_error_handler_func_t,
```

Comment:
We are using the error handler for other things than just client reset, so we still need this.

Reply:
With regard to this, it is my understanding that if a `DiscardLocal` client reset is enabled, this error handler will not be triggered,  but instead the `before/after_client_reset` methods will be invoked as needed.

This method will be invoked for Client reset only if 1) Manual mode was selected or 2) If DiscardLocal failed ...but @ironage can probably confirm?

Reply:
@cmelchior yes that is correct 👍

Reply:
ops, my bad, I must have deleted the wrong line...